pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -302,7 +302,7 @@ func (p *PriorityQueue) Add(pod *v1.Pod) error {
 }
 
 // isPodBackingoff returns true if a pod is still waiting for its backoff timer.
-// If this returns true, the pod should not be re-tried.
+// If this returns true, the pod should be re-tried.
 func (p *PriorityQueue) isPodBackingoff(podInfo *framework.QueuedPodInfo) bool {
```

Comment:
return `false` means that pod is still backing off, and hence should be in the `backoffQ`, only it has passed the backingoff period, the pod could be added back to `activeQ` and retry.